app/services/create_sponsorship.rb

```diff
@@ -0,0 +1,34 @@
+class CreateSponsorship
+
+  attr_reader :sponsorship, :sponsor, :orphan
```

Comment:
do we need the `attr_reader` on services? If we use them in code (eg in controllers) we might just complicate things.
Or they are here only for debugging?
Shouldn't services be used only to perform a set of related tasks and nothing else?